lib/config/migration.ts

```diff
@@ -517,6 +517,17 @@ export function migrateConfig(
         }
       } else if (key === 'binarySource' && val === 'auto') {
         migratedConfig.binarySource = 'global';
+      } else if (key === 'azureAutoComplete') {
+        if (migratedConfig[key]) {
+          migratedConfig.automerge = true;
```

Comment:
```suggestion
```

I think this should be removed, as this was a bug to use without `automerge=true`

Reply:
For Azure, it forces automerge no matter what value of this option is. Though, for GitLab it respected `automerge` value. I tried to make migration preserve current behavior, that's why I handle `azureAutoComplete` and `gitLabAutomerge` in separate if-branches.

Reply:
I think not respecting `automerge` is a bug and we should fix it. If It's intended, gitlab should ignore it too.

@rarkis @JamieMagee WDYT?

Reply:
It should require both to be ready enabled to work